libs/portals/admin/ids-admin/src/screens/Client/CreateClient/CreateClient.action.ts

```diff
@@ -69,6 +70,36 @@ export const createClientAction: WrappedActionFn = ({ client }) => async ({
   }
 
   const { data } = result
+
+  let idAvailableOnAllEnvs = false
```

Comment:
I think you should go at this another way. I would skip this availability check on the client and make the check within the `createClient()` in the `clients.service.ts`. That way you don't have to go back and forth with requests and you do not have to provide special logic in the graphql layer for the archived check, that would just happen within the service.

Reply:
I had this discussion with @saevarma and we decided on this approach. I ran into some issues with the approach that you are suggesting where the client could not determine why the request was failing, if it was because of a duplicate id or because of some other issue or what environment was causing the issue. I know that there are workarounds for this, but at that time with a little time left in the sprint we decided on this approach.